src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1588,11 +1588,10 @@ public async Task<JObject> InvokeMethod(ArraySegment<byte> valueTypeBuffer, int
             return await CreateJObjectForVariableValue(retDebuggerCmdReader, varName, false, -1, false, token);
         }
 
-        public async Task<JObject> InvokeMethodInObject(int objectId, int methodId, string varName, CancellationToken token)
+        public async Task<JObject> InvokeMethodInObject(int objectId, int methodId, bool isValueType, string varName, CancellationToken token)
```

Comment:
Is there a reason not to make the first argument a DotnetObjectId then check isValueType in the body rather than passing two arguments?

Reply:
No, we can change.